attractions/breadcrumbs/breadcrumbs.scss

```
@@ -0,0 +1,13 @@
+@use '../_variables' as vars;
+
+nav {
+  display: flex;
+  align-items: center;
+  flex: 1;
+  margin: 0 2em;
+  font-weight: vars.$thin-font-weight;
+
+  :global(.node) {
```

This might leak inside the custom segments
```suggestion
  & > :global .node {
```

---

That was kinda the point, to be able to give the `node` class to a slot

---

Yeah, this still provides this option, but only for direct children, which is where you'd want it